aptos-move/framework/aptos-framework/doc/aptos_coin.md

```diff
@@ -4,7 +4,7 @@
 # Module `0x1::aptos_coin`
 
 This module defines a minimal and generic Coin and Balance.
-modified from https://github.com/move-language/move/tree/main/language/documentation/tutorial
+modified from https://github.com/move-langauge/move/tree/main/language/documentation/tutorial
```

Comment:
I think this introduces a typo and resulting 404 ("langauge" should be "language"):
Correct - https://github.com/move-language/move/tree/main/language/documentation/tutorial
Incorrect - https://github.com/move-langauge/move/tree/main/language/documentation/tutorial

Reply:
Fixed. Thank you so much for spotting this!!! I must be having a bad day...

Reply:
My pleasure, my friend. I am happy to help.